### PR TITLE
Mass Lavaland Nerfing : No Mercy Edition

### PR DESCRIPTION
**Pull request by Arzion5**
:cl: Arzion
tweak: Lavaland has been nerfed in multiple aspects, mostly regarding the creatures. No more "M-MUH D-DIFFICULTY!!!" for you.
tweak: Meat hook damage reduced to 10 from 25. Meat hook stun reduced to 1 second from 3.
balance: You can no longer transform into an inferior ashwalker when drinking the Dragon's Blood potion anymore.
tweak: Lava Staff damage reduced to 15 from 25. Cooldown for creating lava tiles has been increased to 20 seconds from 10 seconds.
tweak: The gods have realized that the blood-drunk miner is still really just a human, but still felt generous enough to reduce its health to 400 from 900, double the amount of health that a regular human has.
tweak: The nerf hammer has finally come crashing down on megafauna, reducing their health to 1500 from 2500 and damage reductions to their melee attacks (not their special abilities!) and in the case of Colossus and Hierophant, their projectile attacks too. Legion (the megafauna) had its health reduced to 550 from 800. Swarmer Beacon had its health reduced to 300 from 750.
tweak: The nerf hammer has no mercy. Regular fauna is affected too. Goliath health has been reduced to 200 from 300, along with its unique ancient variant having its health reduced to 300 from 400. Goliath damage reduced to 20 from 25. Tentacle stun reduced to 5 seconds from 10.
tweak: Legion (the mob) had its health slightly reduced to 60 from 75, while legion skull damage has been reduced to 8 from 15 (the skulls bite only). The unique variant of the Legion had its health reduced to 100 from 120. Legion's skull spawn delay has been increased, which means you should be able to fight more Legions at a time without being overwhelmed by skulls.
tweak: Watcher's health has been halved to 100 from 200.  Damage reduced to 12 from 15. Magmawing health reduced to 140 from 215. Icewing health reduced to 120 from 170. 
tweak: The tendrils are not safe either. Necropolis tendril health has been reduced to 175 from 250, and the rate at which fauna is spawned from the tendrils has been reduced, taking 60 seconds to spawn a creature instead of 30.
/:cl:

Beg me for mercy.

### CODE REVIEW

**Comment by ghost** [comment]
fast merge -head maintainer

File: hippiestation/code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/obj/item/gun/magic/hook
+	force = 10
+/obj/item/projectile/hook
+	damage = 10 //Wait a second, you're just using the same hook and dragged them!
+	knockdown = 10
+
+/obj/item/dragons_blood/attack_self(mob/living/carbon/human/user)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mining/lavaland/necropolis_chests.dm]: Excuse me sir but you forgot to put this in the changelog.

Comment (Arzion5) [reply hippiestation/code/modules/mining/lavaland/necropolis_chests.dm]: Already there. "tweak: Meat hook damage reduced to 10 from 25. Meat hook stun reduced to 1 second from 3."

Comment (GuyonBroadway) [reply hippiestation/code/modules/mining/lavaland/necropolis_chests.dm]: No the dragons blood changes you Nimby!! You read from the commented line downwards you blithering chunderbucket!

File: hippiestation/code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		if(3)
+			to_chat(user, "<span class='danger'>You feel like you could walk straight through lava now.</span>")
+			H.weather_immunities |= "lava"
+
+	playsound(user.loc,'sound/items/drink.ogg', rand(10,50), 1)
+	qdel(src)
+
+/obj/item/lava_staff
+	force = 15
+	var/create_cooldown = 600 //Jesus Christ, why would you give a miner such a powerful tool that allows you to create impassable lava floors that deal massive burn damage?
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mining/lavaland/necropolis_chests.dm]: remove the var/

File: hippiestation/code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (Arzion5) [reply hippiestation/code/modules/mining/lavaland/necropolis_chests.dm]: "balance: You can no longer transform into an inferior ashwalker when drinking the Dragon's Blood potion anymore."

File: hippiestation/code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/mob/living/simple_animal/hostile/megafauna/hierophant
+	health = 1500
+	maxHealth = 1500
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+
+/obj/effect/temp_visual/hierophant/chaser
+	var/damage = 7
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm]: remove the var/

File: hippiestation/code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+/obj/effect/temp_visual/hierophant/blast
+	var/damage = 7
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm]: remove the var/

File: hippiestation/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+	maxHealth = 200
+	health = 200
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient
+	maxHealth = 300
+	health = 300
+
+/obj/effect/temp_visual/goliath_tentacle/proc/trip()
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: remove /proc

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]
AND FOR CRYING OUT LOUD, TEST ALL THIS SHIT!!

**Comment by ghost** [comment]
Test it all out.
I like the nerfs though.
+1

**Comment by Jujumatic** [comment]
Tentacle stun reduced to 5 seconds from 10.

YOU ARE GREATER THAN JESUS THANK YOU FINALLY

**Comment by Carbonhell** [comment]
>nerfing normal enemies
git gud

**Comment by Arzion5** [comment]
@Carbonhell No u >:(

**Comment by Arzion5** [comment]
All tested, it's all okay, good, comfy and kosher

**Comment by xTrainx** [comment]
the absolute state of hippiestation

**Comment by Arzion5** [comment]
Beg for mercy @xTrainx

**Comment by KayeArray** [comment]
yeah halo 2 brutes were cooler anyway

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by Jujumatic** [comment]
"from Arzion5:Probably-makes-Lavaland-not-be-such-a-piece-of-shit-gamebreaker-holy-fuck"

I agree also can we pls merge now before someone comes and says NO and codegrudges insane buffs into the lavaland mobs :(

**Comment by Carbonhell** [comment]
So basically even an armless dude can mine safely
-1
***-1***

**Comment by GuyonBroadway** [comment]
Your -1 lost its power when u quit being headcoder.

**Comment by Pyko1** [comment]
NO, time to grudgecode lavaland mobs attack and move 10 times faster for balance.

**Comment by Jujumatic** [comment]
Balance: Goliaths have had their stun-time increased to 5 minutes, up from 10 seconds.
Tweak: All mobs can now move 50 tiles per tick
Tweak: All mobs have had their health increased by 500% for "balancing" reasons

REEEEEEEEEEEEEEEEEEEEEE

**Comment by Carbonhell** [comment]
Time to become headcoder again just to -1 this pr

**Comment by ghost** [comment]
-1 stop being shit

**Comment by Jujumatic** [comment]
+1 fix the server lag and maybe we can stop being shit

**Comment by praisenarsie** [comment]
no one saw that

**Comment by doobiedoo23** [comment]
@Carbonhell that wud imply u  have the ability to get headcoder again
BOO MBTFO

**Comment by doobiedoo23** [comment]
also youre actually limp dicked fuckers just remoev lavaland entirely lol

**Comment by JohnMcLaime** [comment]
Since you are moving to new box, up lavaland bombcap, it is just 15 30 60 currently.

**Comment by GuyonBroadway** [comment]
That'd be outside the pr scope John

**Comment by Pyko1** [comment]
So basically every lavaland mob nerfed in multiple ways, I really fail to see the point.
Lavashit is only really challenging to lavaliggers anyways, and this will just prolly make them die even faster since bored miners will be knocking their front door and face even faster with kinetic accelerators.

I mean this PR and the roundstart plasma cutter for ripley with voucher PR combined, why not just spawn all the lavaland ore at mining room aboard the station on round start?

**Comment by Arzion5** [comment]
Lavaland, as of currently, is a mess where people think that if you get tendril stunned for 10 seconds and subsequently mauled, or instantly disappearing for accidentally stepping into a chasm, or being turned to ash or gibbed by megafauna (while since the tech webs rework, you have to wait for some time to produce t4 components to upgrade cloners so that you can be auto-cloned after dying, while at the same time it is unlikely that you'll have your brain recovered by anyone in the case you're gibbed), is challenging, in spirit with the "MOST HOSTILEST PLACE IN THE GAME" and not just extremely annoying. Rewards are completely based on RNG and range from useful and neat (things like the god eye [X-ray and night vision, essentialy complete 7-tile vision, but makes you permanently vulnerable to flashes], the hardsuits, tarot cards) to very shitty (things like soulstones [you aren't given constructs with it], cursed katana, diamond pickaxe). However, I can compromise so that no mob damages will be reduced (except for colossus death bolts), but health reductions will still stay as they are, seeing as fauna and megafauna turn from challenging to very annoying as you become more skilled at surviving in Lavaland, and the megafauna are really just damage sponges.

**Comment by praisenarsie** [comment]
bro are you retarded, just fucking use your arrowkeys to move like lmao 😂 just move away 🚶 just walk away from the tendril ⛺️ bro just smash that arrowkey button 👉  💥 ▶️ like real shit!?

**Comment by geylard** [comment]
first the R&D change to make it built for retards and now mining too? kill all coders before ss13 devolves into a game of pong. 

also goliath stun time is important as it encourages miners to work together and actually adds danger. just because you get grabbed you're not 100% certain to die if you're alone either. 

I don't know what you're talking about with megafauna being damage sponges either, they're literally supposed to be these big bad bosses that should take a while to beat. if you lowered their health then they could easily be farmed with mechs and the blood-drunk miner's thingamadoodle before appreciating the changes in their attack patterns. I mean shit, players like AIIA and roma can already kill them alone frequently.

**Comment by JohnMcLaime** [comment]
There is no point to "farm" megafauna if you have a mech, if you have a mech you have basically all you need for station wide genocide/protecting the disk and being unstunnable.
The loot they drop isnt worth wasting 20 mins (if you are fast/lucky with miners/ore placement) to get mech and then waste extra 5-10 mins getting to the boss and combat mechs cant have plasma cutter (can have rocket launcher but not shitty mining laser?) and luring it to open area to kill it. It isnt worth spending time to order autorifles with trashbags and getting ballistic weapon research for cheap mags. It isnt worth the time getting points to upgrade your KA, same with stock KA.

And they have been balanced by tg, where they still have plenty of 1 hit gg no re weapons, where you can make maxcaps out of all stock ttvs in 5 mins. Combat oriented loot isnt much better than onstation one (thermal eye is bugged and even if it wasnt, it would be still tied to rng drop) and generally, counters to the loot you spent shitton of time getting, can be aqquired in less than 5 mins. As for meme loot, is it worth the time getting to it?

**Comment by geylard** [comment]
maybe if the coders added lavaland mechanics and items instead of trying to take them away. GIVE ME MY CHAOS EMERALDS. GIVE ME MY DORFS.

**Comment by Jujumatic** [comment]
With the way lavaland is there's no point in even going there as you can get better items from the station in half the time, you're guaranteed to get them and you don't have the stupid retard danger of ooh muh random instant death penalty for teh lulz ecks dee xxddddD. If you really want that to stay in lavaland then nobody is even going to even work on it anymore because nobody is going to use lavaland anymore. It's supposed to be high risk = high reward not super high risk = most likely shit rng reward and if you're lucky maybe an ok reward at best but still nothing compared to the god tier shit you can get on the station in the same amount of time.

**Comment by Jujumatic** [comment]
This comments section needs to be locked so only people who've had some time on lavaland can comment on this PR, fuck.

**Comment by geylard** [comment]
"FUCK RNG"

I almost always see a miner on lavaland. There's literally never a, "lol random death xdddD" thing on lavaland, megafauna can be tracked through the GPS and tentacles have a couple second delay for you to get away from them. Instead of making lavaland boring/easy as fuck for players who haven't done lavaland a million times to the point of it only being used as a potential way to get items, how about you FILTHY, DISGUSTING CODERS add more features/items instead of removing the risks of lavaland and nerfing the items as mentioned in the PR? All these nerfs will do is make it appealing to A. Unrobust turbo nerds and B. People trying to farm items so they can kill tendrils then run back to the station as fast as possible to show off their goods. Also as long as lavaland has unique cool items, then it's worth going to. The appeal of lavaland is the challenge, mysteries and the unique loot that can be gained through COMBAT, not "GET THE "BEST" MOST POWERFUL ITEM IN THE SHORTEST AMOUNT OF TIME" because by that theory, why the fuck isn't R&D, xenobiologist, toxins, cargo technician and warden more popular jobs than mining? 

I think you're all just fucking scrubs too. Even as "COMPLETELY BULLSHIT" as lavaland is currently, it's still completely viable to go to lavaland, kill a few tendrils then get back within 15 minutes if you're truly as not shit as you say. I also think the current difficulty is important to lavaland spawns and without the difficulty some of those roles will become increasingly gay.

The only thing that can be improved upon by simply removing/nerfing shit is removing KA upgrade disks from lavaland chests.

 Literally kill everyone in this PR.

**Comment by Arzion5** [comment]
If the PR gets merged but it turns out that it has negative consequences on Lavaland gameplay as a whole, then I will revert the changes myself

**Comment by Swagile** [comment]
- 1

nothing in lavaland has to be fixed because its extremely rare, due to how shit the rng is and how fast the rounds go, for anything in lavaland to ruin the round

**Comment by tortellinitony** [comment]
\-1 Lavaland isn't meant to be easy!

**Comment by Iamgoofball** [comment]
if tg players can handle this you dipshits can too

git gud

**Comment by Iamgoofball** [comment]
ps.

this PR is hippie admitting it's less robust than /tg/

**Comment by Manly-Dwarf** [comment]
-1
Lavaland items like the meathook probably should get a nerf, but you made some good decisions and them some hilariously poor ones.
How about you speed up mining instead of making mining worse?

**Comment by praisenarsie** [comment]
yeah i knew this was going to be disliked

arzion, git gud B)

**Pull request closed without merging**